lib/orderbook/OrderBook.ts

```diff
@@ -80,18 +80,24 @@ class OrderBook {
     return this.addOwnOrder(order);
   }
 
-  public addMarketOrder = async (_order: orders.OwnOrder) => {
-    // TODO: implement
+  public addMarketOrder = async (_order: orders.MarketOrder): Promise<matchingEngine.OrderMatch[]> => {
```

Comment:
the `_` in `_order` can be removed. It was used only when the argument wasn't in use, to relax the compiler.